iOS Client/Models/MBDevotion.swift

```diff
@@ -9,9 +9,14 @@
 import Foundation
 
 
-struct MBDevotion: Codable {
+struct MBDevotion: Codable, Detailable {
     var date: String
     var text: String
     var verse: String
     var verseText: String
 }
+
+struct LoadedDevotion: Codable, Detailable {
```

Comment:
Hmmm, I don't think we should have a distinction between these two types

Reply:
As per discussion, I was wrong about this point. We absolutely need two different structs. One that defines the contract between us and the external data source, and another that is our internal domain model.

They should be independent  (hence, why our domain model should wrap the contract model but just copy over its fields) so that we can insulate our app domain from changes to the external contract. For many changes, we can just update our conversion function that converts the contract model to our domain model.